addons/sale_project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -10,34 +10,32 @@
+                <table class="w-100 table table-striped">
                     <thead>
                         <tr>
-                            <th style="width: 40%">Sales Order Items</th>
-                            <th style="width: 20%; text-align: right;">Sold</th>
-                            <th style="width: 20%; text-align: right;">Delivered</th>
-                            <th style="width: 20%; text-align: right;">Invoiced</th>
+                            <th>Sales Order Items</th>
+                            <th class="text-end">Sold</th>
+                            <th class="text-end">Delivered</th>
+                            <th class="text-end">Invoiced</th>
                         </tr>
                     </thead>
                     <tbody>
                         <tr t-foreach="state.data.sale_items.data" t-as="sale_item" t-key="sale_item.id">
                             <t t-set="uom_name" t-value="sale_item.product_uom and sale_item.product_uom[1]"/>
                             <td>
                                 <t t-set="sol_name" t-value="sale_item.display_name"/>
-                                <a t-if="sale_item.action" class="o_rightpanel_button"
-                                    t-on-click="() => this.onProjectActionClick(sale_item.action)"
-                                >
+                                <a t-if="sale_item.action" class="o_rightpanel_button" href="#" t-on-click="() => this.onProjectActionClick(sale_item.action)">
                                     <t t-esc="sol_name"/>
                                 </a>
                                 <t t-else="" t-esc="sol_name"/>
                             </td>
-                            <td align="right"><t t-esc="formatValue(sale_item.product_uom_qty, uom_name)"/> <t t-esc="uom_name"/></td>
-                            <td align="right"><t t-esc="formatValue(sale_item.qty_delivered, uom_name)"/> <t t-esc="uom_name"/></td>
-                            <td align="right"><t t-esc="formatValue(sale_item.qty_invoiced, uom_name)"/> <t t-esc="uom_name"/></td>
+                            <td class="text-end"><t t-esc="formatValue(sale_item.product_uom_qty, uom_name)"/> <t t-esc="uom_name"/></td>
+                            <td class="text-end"><t t-esc="formatValue(sale_item.qty_delivered, uom_name)"/> <t t-esc="uom_name"/></td>
+                            <td class="text-end"><t t-esc="formatValue(sale_item.qty_invoiced, uom_name)"/> <t t-esc="uom_name"/></td>
                         </tr>
-                        <tr class="o_rightpanel_nohover" t-if="state.data.sale_items.total &gt; state.data.sale_items.data.length">
-                            <td>
-                                <a class="o_rightpanel_button text-bold" t-on-click="onLoadSalesOrderLinesClick">
+                        <tr class="o_rightpanel_nohover border-0" t-if="state.data.sale_items.total &gt; state.data.sale_items.data.length">
```

Comment:
Let's render something to replace the [Load more] btn when it's not visible, otherwise tables will be to close

https://user-images.githubusercontent.com/8558351/188656354-006c998e-4916-452b-b023-94b7af54ac9a.mov

Reply:
You can check that too